starter-kits/credential-registry/server/tob-api/api_v2/serializers/search.py

```diff
@@ -233,7 +233,7 @@ class Meta(CredentialSerializer.Meta):
             "latest",
             "revoked",
             "revoked_date",
-            "wallet_id",
+            "credential_exchange_id",
```

Comment:
Are the change sin this file going to affect our UI? E.g.: we need to expect a different returned property name?

Reply:
@esune I don't think so. I replaced all references in entire repo and there were no references in the client.